theSrc/scripts/RectPlot.js

```diff
@@ -147,12 +144,6 @@ class RectPlot {
       pointSize: config.trendLinesPointSize,
     }
 
-    // TODO configure
```

Comment:
These are actually your changes which for some reason got confused after I rebased